lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -0,0 +1,20 @@
+module ShopifyAPI
+  class FulfillmentOrder < Base
+    def self.find(scope, *args)
+      if scope == :all
+        order_id = args.first&.dig(:params, :order_id)
+        raise ShopifyAPI::ValidationException, "'order_id' is required" if order_id.blank?
+
+        order = ::ShopifyAPI::Order.new(id: order_id)
+        order.fulfillment_orders(args.first[:params].except(:order_id))
```

Comment:
I don't think we need to support the case of `all` here. If a user wants to get all `fulfillment_orders` for an `order`, they can call `order.fulfillment_orders`. Support calling `.all` with a param `order_id` seems redundant IMO.

Reply:
Given that FulfillmentOrder can be queried as a top-level resource with `find` it would be unexpected from an ActiveResource implementation to not support `all` in some meaningful way.